client/src/Routes.jsx

```diff
@@ -1,17 +1,41 @@
+/* eslint-disable react/no-unused-state */
 import React from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import App from './App';
-import NavBar from './components/UI/NavBar';
+import { Provider } from './api/userContext';
 
-const PacmanRouter = () => (
-  <div>
-    <NavBar isLoggedIn />
-    <BrowserRouter>
-      <Switch>
-        <Route path="/" component={App} />
-      </Switch>
-    </BrowserRouter>
-  </div>
-);
+class PacmanRouter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
use class variables

Reply:
I did used the class variable, but the functions in the state were not accessible in other components through that. so I wrapped it in constructor.